--- zip/BUILD.bazel
+++ .build/zip.BUILD.bazel
@@ -1,36 +1,6 @@
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
-
+# buildifier: disable=module-docstring
 package(default_visibility = ["//visibility:public"])
 
-bzl_library(
-    name = "defs",
-    srcs = ["defs.bzl"],
-    deps = [
-        "//zip/private:zip",
-    ],
-)
-
-bzl_library(
-    name = "assets",
-    srcs = [
-        "assets.bzl",
-    ],
-    deps = [
-        "@bzlparty_tools//lib:platforms",
-    ],
-)
-
-bzl_library(
-    name = "repositories",
-    srcs = ["repositories.bzl"],
-    deps = [
-        ":assets",
-        "//utils:repositories",
-        "@bzlparty_tools//lib:github",
-        "@bzlparty_tools//lib:platforms",
-    ],
-)
-
 toolchain_type(
     name = "zip_toolchain_type",
 )
